jkube-kit/resource/helm/src/main/java/org/eclipse/jkube/kit/resource/helm/oci/OCIRegistryEndpoint.java

```diff
@@ -53,6 +55,12 @@ public String getManifestUrl(Chart chart) {
     return String.format("%s/%s/manifests/%s", apiV2Url, chart.getName(), chart.getVersion());
   }
 
+  public String getOCIRegistryHost() {
+    String hostStr = baseUrl.getHost();
+    String portStr = baseUrl.getPort() > 0 ? (":" + baseUrl.getPort()) : EMPTY;
+    return String.format("%s%s", hostStr, portStr);
```

Comment:
**nit**

This is extremely verbose and complicated to read. You're mixing template strings with ternaries and string concatenation.

Something like:
```java
final StringBuilder host = new StringBuilder(baseUrl.getHost());
if (baseUrl.getPort() > 0 && baseUrl.getPort() != 80 && baseUrl.getPort() != 443) {
  test.append(":").append(baseUrl.getPort());
}
System.out.println(test.toString());
```

is easier to understand and doesn't mix 3 different ways of concatenating strings.